kubernetes-client-openapi/src/main/java/io/micronaut/kubernetes/client/openapi/config/KubeConfigLoader.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.kubernetes.client.openapi.config;
+
+import io.micronaut.core.annotation.NonNull;
+
+/**
+ * The loader for kube config file.
```

Comment:
describe the semantics of the methods of this class. Is it loaded once? Is it loaded repeatedly? Is it thread safe?

Reply:
Haven't done this yet. Will do.

Reply:
Done